react/Empty/index.jsx

```diff
@@ -5,9 +5,16 @@ import { Text, MainTitle } from '../Text'
 import Icon, { iconPropType } from '../Icon'
 import styles from './styles.styl'
 
-export const Empty = ({ icon, title, text, children, className }) => {
+export const Empty = ({
+  icon,
+  title,
+  text,
+  children,
+  className,
+  ...others
+}) => {
   return (
-    <div className={cx(styles['c-empty'], className)}>
+    <div className={cx(styles['c-empty'], className)} {...others}>
```

Comment:
We have to be careful, with React, we can have a lot of warning by passing non-dom attributes to a react component, what about a specific props named `htmlAttributes` containing all attributes we want to pass to the component? And we would be able to expose that way to all other components

Reply:
We had already had this discussion and we agreed to go this way as other UI libs are doing the same. 

The App has to be careful yes. Not the lib ;).

Reply:
See https://github.com/cozy/cozy-ui/issues/821